projects/angular-auth-oidc-client/src/lib/config/openid-configuration.ts

```diff
@@ -1,43 +1,147 @@
 import { LogLevel } from '../logging/log-level';
 
 export interface OpenIdConfiguration {
+  /**
+   * The url to the Security Token Service (STS) server.
+   * This field is required.
+   */
   stsServer?: string;
+  /** Override the default STS wellknown endpoint postfix. */
   authWellknownEndpoint?: string;
+  /** The redirect URL defined on the STS. */
   redirectUrl?: string;
+  /**
+   * The Client MUST validate that the aud (audience) Claim contains its client_id value
+   * registered at the Issuer identified by the iss (issuer) Claim as an audience.
+   * The ID Token MUST be rejected if the ID Token does not list the Client as a valid audience,
+   * or if it contains additional audiences not trusted by the Client.
+   */
   clientId?: string;
+  /**
+   * 'code', 'id_token token' or 'id_token' Name of the flow which can be configured.
+   * You must use the 'id_token token' flow, if you want to access an API or get user data from the server.
+   * The access_token is required for this, and only returned with this flow.
+   */
   responseType?: string;
+  /**
+   * This is this scopes which are requested from the server from this client.
+   * This must match the STS server configuration.
```

Comment:
Can you append this:

The openid scope is required. This offline_access scope can be requested when using refresh tokens but this is optional and some STS do not support this or recommend not requesting this even when using refresh tokens in the browser.

Reply:
This is now corrected.